src/components/sidebar/Sidebar.tsx

```diff
@@ -82,24 +86,17 @@ export default function Sidebar() {
   const [hoveredButton, setHoveredButton] = useState<string | null>(null);
 
   const { isAuthenticated } = useAuth();
-  const { voyageTeamMembers } = useUser();
-
-  const isActive = useMemo(() => {
-    if (voyageTeamMembers.length === 0) {
-      return false;
-    }
-    return voyageTeamMembers.some(
-      (member) => member.voyageTeam.voyage.status.name === "Active",
-    );
-  }, [voyageTeamMembers]);
-
-  const isVoyageStarted: boolean = isAuthenticated && isActive;
-
-  const currentVoyageTeam = voyageTeamMembers.find(
-    (voyage) => voyage.voyageTeam.voyage.status.name === "Active",
+  const user = useUser();
+  const voyageTeamAdapter = resolve<VoyageTeamClientAdapter>(
```

Comment:
Can we define these adapters in separate files? This approach would make the codebase more scalable.
For example:

```
export const voyageAdapters = {
  voyageTeamAdapter: resolve<VoyageTeamClientAdapter>(TYPES.VoyageTeamClientAdapter)
} as const;
```

We could organize the files like this:

-resolve
--voyage.ts
--another_adapter.ts
....

What do you think?